rule/fetcher_default_test.go

```diff
@@ -67,3 +68,58 @@ func TestFetcher(t *testing.T) {
 		})
 	}
 }
+
+func TestFetcherDefault_Fetch_formats(t *testing.T) {
```

Comment:
This is a nitpick, we're not using underscores in test names - could you rename this to `TestFetcherDefaultFetchFormats`? :)

Reply:
Oops, I thought you are using "generic" Go convention `TestStruct_Method`. Ok, I will rename this :)